examples/simpleapp/readme.md

```diff
@@ -1,7 +1,7 @@
 **Simple App**
 ---------
 
-Simple app is a Java application that allows to run a pure Sidechain Node without any custom logic.
+Simple app is a Java application that allows to run a pure Sidechain Node without any custom logic like custom boxes and transactions. Forward and Backward transactions are available from box.
```

Comment:
`Forward and Backward transactions are available from box.` -> `Forward and Backward transfers are available out of the box.`

Reply:
done